recipe/build.sh

```diff
@@ -1,20 +1,28 @@
 #!/bin/bash
 
+if [[ $target_platform == linux* ]]; then
+  ln -s -t "${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64/" $PREFIX/lib/libexpat*
```

Comment:
Looking at your commits to try to understand this I realized I would never get this right :smile:  Thanks @isuruf!

However, it seems bad IMO that we need this instead of just specified expat in the dependency list.